modules/core/client/views/careers.client.view.html

```diff
@@ -10,15 +10,32 @@ <h1 class="header center orange-text">Careers</h1>
 			    <h5 class="header col s12 light">Available Positions</h5>
 			</div>
 
-			<div class="row center">
-		  		<a id="contact-us-button" class="btn-large waves-effect waves-light orange" ng-click="openCareerApplication();">Apply Now</a>
-			</div>
-
 		</div>
 
 	</div>
 
-	<div class="lightbox ng-hide" ng-show="careerApplicationVisible">
+  <div class="row left">
+    <ul class="left jobsList">
+      <li class="jobList-item" ng-repeat="job in Jobs">
+        <div class="row">
+          <h5 class="col s12 light">{{job}}</h5>
+        </div>
+        <div class="row">
+          <span class="col s12 light">Description of Job goes here blah blah blah blah blah b lbkajasdfkjhasdkjfhaskjdhfksajdhfaksjdhfkajshfdksajhdfajksdfhasjkdfhajksdhfsdjkfhajkshdf
+                                      blah blah b lbkajasdfkjhasdkjfhaskjdhfksajdhfaksjdhfkajshfdksajhdfajksdfhasjkdfhajksdhfsdjkfhajkshdf
+                                      blah blah b lbkajasdfkjhasdkjfhaskjdhfksajdhfaksjdhfkajshfdksajhdfajksdfhasjkdfhajksdhfsdjkfhajkshdf....</span>
+        </div>
+        <div class="row">
+          <span class="right col s1 light">
+            <a id="contact-us-button" class="btn-large waves-effect waves-light orange" ng-click="openCareerApplication();">Apply Now</a>
+          </span>
+        </div>
+      </li>
+    </ul>
+  </div>
+
+
+	<div class="lightbox-app ng-hide" ng-show="careerApplicationVisible">
```

Comment:
To add a close button, include "<i class="material-icons close-button" ng-click="closeCareerApplication();">close</i>" in the 'lightbox-app' div